--- docker-entrypoint.sh	2018-02-09 21:19:03.687392177 -0500
+++ entrypoint_modif	2018-02-09 21:18:44.401466364 -0500
@@ -93,6 +93,10 @@
 		"${uniqueEnvs[@]/#/WORDPRESS_}"
 		WORDPRESS_TABLE_PREFIX
 		WORDPRESS_DEBUG
+		SITE
+		USER
+		PASS
+		EMAIL
 	)
 	haveConfig=
 	for e in "${envs[@]}"; do
@@ -234,6 +238,22 @@
 $mysql->close();
 EOPHP
 	fi
+    
+    if [ ! -d wp-content/plugins/nginx-helper ]; then
+
+	echo >&2 "WordPress not configured..."
+            
+    wp core install --url=$SITE --title="Site Web" --admin_user=$USER --admin_password=$PASS --admin_email=$EMAIL --allow-root
+
+	wp plugin install wp-clone-by-wp-academy --activate --allow-root
+
+	wp plugin install nginx-helper --activate --allow-root
+
+	wp plugin install redis-cache --activate --allow-root
+	
+	chown www-data:www-data -R ./
+	
+	fi
 
 	# now that we're definitely done writing configuration, let's clear out the relevant envrionment variables (so that stray "phpinfo()" calls don't leak secrets from our code)
 	for e in "${envs[@]}"; do
