--- docker-entrypoint.sh	2018-01-14 21:46:48.927080028 -0500
+++ entrypoint_modif	2018-01-14 21:46:22.835417487 -0500
@@ -241,4 +241,12 @@
 	done
 fi
 
+wp core install --url=$site --title="Site Web" --admin_user=$user --admin_password=$pass --admin_email=$email
+
+wp plugin install wp-clone-by-wp-academy --activate
+
+wp plugin install nginx-helper --activate
+
+wp plugin install redis-cache --activate
+
 exec "$@"
