--- docker-entrypoint.sh	2018-03-19 21:26:26.396844356 -0400
+++ entrypoint_modif	2018-03-19 21:26:14.664935066 -0400
@@ -93,6 +93,10 @@
 		"${uniqueEnvs[@]/#/WORDPRESS_}"
 		WORDPRESS_TABLE_PREFIX
 		WORDPRESS_DEBUG
+		SITE
+		USER
+		PASS
+		EMAIL
 	)
 	haveConfig=
 	for e in "${envs[@]}"; do
@@ -234,11 +238,50 @@
 $mysql->close();
 EOPHP
 	fi
+    
+    if [ ! -d wp-content/plugins/nginx-helper ]; then
+
+	echo >&2 "WordPress not configured..."
+            
+    wp core install --url=$SITE --title="Site Web" --admin_user=$USER --admin_password=$PASS --admin_email=$EMAIL --allow-root
+
+	wp plugin install nginx-helper --activate --allow-root
+
+	wp plugin install redis-cache --activate --allow-root
+	
+	chown www-data:www-data -R ./
+	
+	sed -i "/WP_DEBUG/a define('RT_WP_NGINX_HELPER_CACHE_PATH', '/tmp/cache');" wp-config.php
+	
+	sed -i "/WP_DEBUG/a define('WP_CACHE_KEY_SALT', '$SITE');" wp-config.php
+	
+	sed -i "/WP_DEBUG/a define('WP_REDIS_HOST', 'redis');" wp-config.php
+		
+	mkdir -p wp-content/uploads/nginx-helper
+
+	fi
+
+echo "Setup msmtp"
+
+file_env 'EMAIL_PW'
+
+echo "account        default" >> /etc/msmtprc
+echo "host           $EMAIL_SERVER" >> /etc/msmtprc
+echo "port           $EMAIL_PORT" >> /etc/msmtprc
+echo "from           $EMAIL_FROM" >> /etc/msmtprc
+echo "user           $EMAIL_USER" >> /etc/msmtprc
+echo "password       $EMAIL_PW" >> /etc/msmtprc
+
+echo 'sendmail_path = "/usr/bin/msmtp -C /etc/msmtprc -t"' > /usr/local/etc/php/conf.d/mail.ini
+
+chmod 400 /etc/msmtprc
+chown www-data:www-data /etc/msmtprc
 
 	# now that we're definitely done writing configuration, let's clear out the relevant envrionment variables (so that stray "phpinfo()" calls don't leak secrets from our code)
 	for e in "${envs[@]}"; do
 		unset "$e"
 	done
+
 fi
 
 exec "$@"
