--- docker-entrypoint.sh	2018-01-29 22:06:57.604348132 -0500
+++ entrypoint_modif	2018-01-29 22:06:53.356372933 -0500
@@ -93,6 +93,10 @@
 		"${uniqueEnvs[@]/#/WORDPRESS_}"
 		WORDPRESS_TABLE_PREFIX
 		WORDPRESS_DEBUG
+		SITE
+		USER
+		PASS
+		EMAIL
 	)
 	haveConfig=
 	for e in "${envs[@]}"; do
@@ -234,6 +238,18 @@
 $mysql->close();
 EOPHP
 	fi
+    
+    if [ ! -d wp-content/plugins/nginx-helper ]; then
+            
+    wp core install --url=$SITE --title="Site Web" --admin_user=$USER --admin_password=$PASS --admin_email=$EMAIL --allow-root
+
+	wp plugin install wp-clone-by-wp-academy --activate --allow-root
+
+	wp plugin install nginx-helper --activate --allow-root
+
+	wp plugin install redis-cache --activate --allow-root
+	
+	fi
 
 	# now that we're definitely done writing configuration, let's clear out the relevant envrionment variables (so that stray "phpinfo()" calls don't leak secrets from our code)
 	for e in "${envs[@]}"; do
